website/cue/reference/services/http_scrape.cue

```diff
@@ -0,0 +1,8 @@
+package metadata
+
+services: http_scrape: {
+	name:     "HTTP scrape"
+	thing:    "a \(name)"
```

Comment:
Should this be `"a \(name)er"`? The part of speech looks like it might be awkward.